octopoes/.ci/docker-compose.yml

```diff
@@ -38,11 +38,40 @@ services:
     command: pytest -x tests/integration
     depends_on:
       - xtdb_multinode
+      - ci_octopoes
     env_file:
       - ".ci/ci-multinode.env"
 
   xtdb_multinode:
-    image: "ghcr.io/dekkers/xtdb-http-multinode:v1.0.4"
+    image: "ghcr.io/dekkers/xtdb-http-multinode:pr-11"
```

Comment:
Shouldn't this become `ghcr.io/dekkers/xtdb-http-multinode:v1.0.6` now that the PR has been merged?